### PR TITLE
[testing] switch to parallel test

### PR DESCRIPTION
**Pull request by ryanking**
running acceptance tests locally...

Before:

```
real	3m18.942s
user	0m48.747s
sys	0m31.052s
```

After:

```
real	0m57.765s
user	0m59.877s
sys	0m43.979s
```

## Test Plan
* [ ] acceptance tests

## References
*

### CODE REVIEW

File: pkg/resources/account_grant_acceptance_test.go [hunk 1]
```diff
@@ -11,7 +11,7 @@ import (
 func TestAccAccountGrant_defaults(t *testing.T) {
 	roleName := acctest.RandStringFromCharSet(10, acctest.CharSetAlpha)
 
-	resource.Test(t, resource.TestCase{
+	resource.ParallelTest(t, resource.TestCase{
```
Comment (cziie) [inline pkg/resources/account_grant_acceptance_test.go:14]: **[tfproviderlint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>AT001: missing CheckDestroy

File: pkg/resources/database_acceptance_test.go [hunk 2]
```diff
@@ -17,7 +17,7 @@ func TestAcc_Database(t *testing.T) {
 	prefix := acctest.RandStringFromCharSet(10, acctest.CharSetAlpha)
 	prefix2 := acctest.RandStringFromCharSet(10, acctest.CharSetAlpha)
 
-	resource.Test(t, resource.TestCase{
+	resource.ParallelTest(t, resource.TestCase{
```
Comment (cziie) [inline pkg/resources/database_acceptance_test.go:20]: **[tfproviderlint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>AT001: missing CheckDestroy

File: pkg/resources/database_grant_acceptance_test.go [hunk 3]
```diff
@@ -40,7 +40,7 @@ func TestAcc_DatabaseGrant(t *testing.T) {
 	roleName := acctest.RandStringFromCharSet(10, acctest.CharSetAlpha)
 	shareName := acctest.RandStringFromCharSet(10, acctest.CharSetAlpha)
 
-	resource.Test(t, resource.TestCase{
+	resource.ParallelTest(t, resource.TestCase{
```
Comment (cziie) [inline pkg/resources/database_grant_acceptance_test.go:43]: **[tfproviderlint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>AT001: missing CheckDestroy

File: pkg/resources/managed_account_acceptance_test.go [hunk 4]
```diff
@@ -22,7 +22,7 @@ func TestAcc_ManagedAccount(t *testing.T) {
 	adminName := acctest.RandStringFromCharSet(10, acctest.CharSetAlpha)
 	adminPass := fmt.Sprintf("A1%v", acctest.RandStringFromCharSet(10, acctest.CharSetAlpha))
 
-	resource.Test(t, resource.TestCase{
+	resource.ParallelTest(t, resource.TestCase{
```
Comment (cziie) [inline pkg/resources/managed_account_acceptance_test.go:25]: **[tfproviderlint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>AT001: missing CheckDestroy

File: pkg/resources/network_policy_acceptance_test.go [hunk 5]
```diff
@@ -20,7 +20,7 @@ func TestAcc_NetworkPolicy(t *testing.T) {
 
 	name := acctest.RandStringFromCharSet(10, acctest.CharSetAlpha)
 
-	resource.Test(t, resource.TestCase{
+	resource.ParallelTest(t, resource.TestCase{
```
Comment (cziie) [inline pkg/resources/network_policy_acceptance_test.go:23]: **[tfproviderlint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>AT001: missing CheckDestroy

File: pkg/resources/network_policy_attachment_acceptance_test.go [hunk 6]
```diff
@@ -18,7 +18,7 @@ func TestAcc_NetworkPolicyAttachment(t *testing.T) {
 	user2 := acctest.RandStringFromCharSet(10, acctest.CharSetAlpha)
 	policyName := acctest.RandStringFromCharSet(10, acctest.CharSetAlpha)
 
-	resource.Test(t, resource.TestCase{
+	resource.ParallelTest(t, resource.TestCase{
```
Comment (cziie) [inline pkg/resources/network_policy_attachment_acceptance_test.go:21]: **[tfproviderlint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>AT001: missing CheckDestroy

File: pkg/resources/resource_monitor_acceptance_test.go [hunk 7]
```diff
@@ -12,7 +12,7 @@ func TestAcc_ResourceMonitor(t *testing.T) {
 	// TODO test more attributes
 	name := acctest.RandStringFromCharSet(10, acctest.CharSetAlpha)
 
-	resource.Test(t, resource.TestCase{
+	resource.ParallelTest(t, resource.TestCase{
```
Comment (cziie) [inline pkg/resources/resource_monitor_acceptance_test.go:15]: **[tfproviderlint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>AT001: missing CheckDestroy

File: pkg/resources/resource_monitor_grant_acceptance_test.go [hunk 8]
```diff
@@ -12,7 +12,7 @@ func TestAccResourceMonitor_defaults(t *testing.T) {
 	wName := acctest.RandStringFromCharSet(10, acctest.CharSetAlpha)
 	roleName := acctest.RandStringFromCharSet(10, acctest.CharSetAlpha)
 
-	resource.Test(t, resource.TestCase{
+	resource.ParallelTest(t, resource.TestCase{
```
Comment (cziie) [inline pkg/resources/resource_monitor_grant_acceptance_test.go:15]: **[tfproviderlint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>AT001: missing CheckDestroy

File: pkg/resources/role_acceptance_test.go [hunk 9]
```diff
@@ -12,7 +12,7 @@ func TestAcc_Role(t *testing.T) {
 	prefix := acctest.RandStringFromCharSet(10, acctest.CharSetAlpha)
 	prefix2 := acctest.RandStringFromCharSet(10, acctest.CharSetAlpha)
 
-	resource.Test(t, resource.TestCase{
+	resource.ParallelTest(t, resource.TestCase{
```
Comment (cziie) [inline pkg/resources/role_acceptance_test.go:15]: **[tfproviderlint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>AT001: missing CheckDestroy

File: pkg/resources/role_grants_acceptance_test.go [hunk 10]
```diff
@@ -108,7 +108,7 @@ func TestAcc_GrantRole(t *testing.T) {
 		),
 	}
 
-	resource.Test(t, resource.TestCase{
+	resource.ParallelTest(t, resource.TestCase{
```
Comment (cziie) [inline pkg/resources/role_grants_acceptance_test.go:111]: **[tfproviderlint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>AT001: missing CheckDestroy

File: pkg/resources/schema_acceptance_test.go [hunk 11]
```diff
@@ -11,7 +11,7 @@ import (
 func TestAcc_Schema(t *testing.T) {
 	accName := acctest.RandStringFromCharSet(10, acctest.CharSetAlpha)
 
-	resource.Test(t, resource.TestCase{
+	resource.ParallelTest(t, resource.TestCase{
```
Comment (cziie) [inline pkg/resources/schema_acceptance_test.go:14]: **[tfproviderlint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>AT001: missing CheckDestroy

File: pkg/resources/schema_grant_acceptance_test.go [hunk 12]
```diff
@@ -19,7 +19,7 @@ func TestAcc_SchemaGrant(t *testing.T) {
 	roleName := strings.ToUpper(acctest.RandStringFromCharSet(10, acctest.CharSetAlpha))
 	shareName := strings.ToUpper(acctest.RandStringFromCharSet(10, acctest.CharSetAlpha))
 
-	resource.Test(t, resource.TestCase{
+	resource.ParallelTest(t, resource.TestCase{
```
Comment (cziie) [inline pkg/resources/schema_grant_acceptance_test.go:22]: **[tfproviderlint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>AT001: missing CheckDestroy

File: pkg/resources/schema_grant_acceptance_test.go [hunk 13]
```diff
@@ -55,7 +55,7 @@ func TestAcc_SchemaFutureGrants(t *testing.T) {
 	roleNameTable := strings.ToUpper(acctest.RandStringFromCharSet(10, acctest.CharSetAlpha))
 	roleNameView := strings.ToUpper(acctest.RandStringFromCharSet(10, acctest.CharSetAlpha))
 
-	resource.Test(t, resource.TestCase{
+	resource.ParallelTest(t, resource.TestCase{
```
Comment (cziie) [inline pkg/resources/schema_grant_acceptance_test.go:58]: **[tfproviderlint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>AT001: missing CheckDestroy

File: pkg/resources/share_acceptance_test.go [hunk 14]
```diff
@@ -20,7 +20,7 @@ func TestAcc_Share(t *testing.T) {
 
 	name := acctest.RandStringFromCharSet(10, acctest.CharSetAlpha)
 
-	resource.Test(t, resource.TestCase{
+	resource.ParallelTest(t, resource.TestCase{
```
Comment (cziie) [inline pkg/resources/share_acceptance_test.go:23]: **[tfproviderlint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>AT001: missing CheckDestroy

File: pkg/resources/stage_acceptance_test.go [hunk 15]
```diff
@@ -11,7 +11,7 @@ import (
 func TestAcc_Stage(t *testing.T) {
 	accName := acctest.RandStringFromCharSet(10, acctest.CharSetAlpha)
 
-	resource.Test(t, resource.TestCase{
+	resource.ParallelTest(t, resource.TestCase{
```
Comment (cziie) [inline pkg/resources/stage_acceptance_test.go:14]: **[tfproviderlint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>AT001: missing CheckDestroy

File: pkg/resources/stage_grant_acceptance_test.go [hunk 16]
```diff
@@ -11,7 +11,7 @@ import (
 func TestAccStageGrant_defaults(t *testing.T) {
 	name := acctest.RandStringFromCharSet(10, acctest.CharSetAlpha)
 
-	resource.Test(t, resource.TestCase{
+	resource.ParallelTest(t, resource.TestCase{
```
Comment (cziie) [inline pkg/resources/stage_grant_acceptance_test.go:14]: **[tfproviderlint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>AT001: missing CheckDestroy

File: pkg/resources/stream_acceptance_test.go [hunk 17]
```diff
@@ -11,7 +11,7 @@ import (
 func TestAcc_Stream(t *testing.T) {
 	accName := acctest.RandStringFromCharSet(10, acctest.CharSetAlpha)
 
-	resource.Test(t, resource.TestCase{
+	resource.ParallelTest(t, resource.TestCase{
```
Comment (cziie) [inline pkg/resources/stream_acceptance_test.go:14]: **[tfproviderlint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>AT001: missing CheckDestroy

File: pkg/resources/table_acceptance_test.go [hunk 18]
```diff
@@ -11,7 +11,7 @@ import (
 func TestAcc_Table(t *testing.T) {
 	accName := acctest.RandStringFromCharSet(10, acctest.CharSetAlpha)
 
-	resource.Test(t, resource.TestCase{
+	resource.ParallelTest(t, resource.TestCase{
```
Comment (cziie) [inline pkg/resources/table_acceptance_test.go:14]: **[tfproviderlint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>AT001: missing CheckDestroy

File: pkg/resources/table_grant_acceptance_test.go [hunk 19]
```diff
@@ -11,7 +11,7 @@ import (
 func TestAccTableGrant_defaults(t *testing.T) {
 	name := acctest.RandStringFromCharSet(10, acctest.CharSetAlpha)
 
-	resource.Test(t, resource.TestCase{
+	resource.ParallelTest(t, resource.TestCase{
```
Comment (cziie) [inline pkg/resources/table_grant_acceptance_test.go:14]: **[tfproviderlint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>AT001: missing CheckDestroy

File: pkg/resources/task_acceptance_test.go [hunk 20]
```diff
@@ -162,7 +162,7 @@ var (
 func TestAcc_Task(t *testing.T) {
 	t.Skip("broken by a change to snowflake")
 
-	resource.Test(t, resource.TestCase{
+	resource.ParallelTest(t, resource.TestCase{
```
Comment (cziie) [inline pkg/resources/task_acceptance_test.go:165]: **[tfproviderlint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>AT001: missing CheckDestroy

File: pkg/resources/user_acceptance_test.go [hunk 21]
```diff
@@ -39,7 +39,7 @@ func TestAcc_User(t *testing.T) {
 	sshkey2, err := testhelpers.Fixture("userkey2")
 	r.NoError(err)
 
-	resource.Test(t, resource.TestCase{
+	resource.ParallelTest(t, resource.TestCase{
```
Comment (cziie) [inline pkg/resources/user_acceptance_test.go:42]: **[tfproviderlint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>AT001: missing CheckDestroy

File: pkg/resources/view_acceptance_test.go [hunk 22]
```diff
@@ -11,7 +11,7 @@ import (
 func TestAcc_View(t *testing.T) {
 	accName := acctest.RandStringFromCharSet(10, acctest.CharSetAlpha)
 
-	resource.Test(t, resource.TestCase{
+	resource.ParallelTest(t, resource.TestCase{
```
Comment (cziie) [inline pkg/resources/view_acceptance_test.go:14]: **[tfproviderlint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>AT001: missing CheckDestroy

File: pkg/resources/view_acceptance_test.go [hunk 23]
```diff
@@ -30,7 +30,7 @@ func TestAcc_View(t *testing.T) {
 func TestAcc_View2(t *testing.T) {
 	accName := acctest.RandStringFromCharSet(10, acctest.CharSetAlpha)
 
-	resource.Test(t, resource.TestCase{
+	resource.ParallelTest(t, resource.TestCase{
```
Comment (cziie) [inline pkg/resources/view_acceptance_test.go:33]: **[tfproviderlint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>AT001: missing CheckDestroy

File: pkg/resources/view_grant_acceptance_test.go [hunk 24]
```diff
@@ -16,7 +16,7 @@ func TestAcc_ViewGrantBasic(t *testing.T) {
 	databaseName := acctest.RandStringFromCharSet(10, acctest.CharSetAlpha)
 	roleName := acctest.RandStringFromCharSet(10, acctest.CharSetAlpha)
 
-	resource.Test(t, resource.TestCase{
+	resource.ParallelTest(t, resource.TestCase{
```
Comment (cziie) [inline pkg/resources/view_grant_acceptance_test.go:19]: **[tfproviderlint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>AT001: missing CheckDestroy

File: pkg/resources/view_grant_acceptance_test.go [hunk 25]
```diff
@@ -40,7 +40,7 @@ func TestAcc_ViewGrantShares(t *testing.T) {
 	roleName := acctest.RandStringFromCharSet(10, acctest.CharSetAlpha)
 	shareName := acctest.RandStringFromCharSet(10, acctest.CharSetAlpha)
 
-	resource.Test(t, resource.TestCase{
+	resource.ParallelTest(t, resource.TestCase{
```
Comment (cziie) [inline pkg/resources/view_grant_acceptance_test.go:43]: **[tfproviderlint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>AT001: missing CheckDestroy

File: pkg/resources/view_grant_acceptance_test.go [hunk 26]
```diff
@@ -59,7 +59,7 @@ func TestAcc_FutureViewGrantChange(t *testing.T) {
 	databaseName := acctest.RandStringFromCharSet(10, acctest.CharSetAlpha)
 	roleName := acctest.RandStringFromCharSet(10, acctest.CharSetAlpha)
 
-	resource.Test(t, resource.TestCase{
+	resource.ParallelTest(t, resource.TestCase{
```
Comment (cziie) [inline pkg/resources/view_grant_acceptance_test.go:62]: **[tfproviderlint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>AT001: missing CheckDestroy

File: pkg/resources/warehouse_acceptance_test.go [hunk 27]
```diff
@@ -17,7 +17,7 @@ func TestAcc_Warehouse(t *testing.T) {
 	prefix := acctest.RandStringFromCharSet(10, acctest.CharSetAlpha)
 	prefix2 := acctest.RandStringFromCharSet(10, acctest.CharSetAlpha)
 
-	resource.Test(t, resource.TestCase{
+	resource.ParallelTest(t, resource.TestCase{
```
Comment (cziie) [inline pkg/resources/warehouse_acceptance_test.go:20]: **[tfproviderlint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>AT001: missing CheckDestroy

File: pkg/resources/warehouse_grant_acceptance_test.go [hunk 28]
```diff
@@ -16,7 +16,7 @@ func TestAcc_WarehouseGrant(t *testing.T) {
 	wName := acctest.RandStringFromCharSet(10, acctest.CharSetAlpha)
 	roleName := acctest.RandStringFromCharSet(10, acctest.CharSetAlpha)
 
-	resource.Test(t, resource.TestCase{
+	resource.ParallelTest(t, resource.TestCase{
```
Comment (cziie) [inline pkg/resources/warehouse_grant_acceptance_test.go:19]: **[tfproviderlint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>AT001: missing CheckDestroy

**Review by edulop91 — APPROVED** [review]
:man_dancing:

**Pull request merged**